pytgcalls/pytgcalls/implementation/group_call_base.py

```diff
@@ -418,8 +418,14 @@ async def set_my_volume(self, volume):
         await self.edit_group_call(volume)
         self._set_volume(uint_ssrc(self.mtproto.my_ssrc), volume / 100)
 
+    get_group_call_members = self.mtproto.get_group_particiants
+
     async def get_my_volume(self):
-        volume = await self.mtproto.get_my_volume()
+        if not self.my_ssrc or not self.my_peer:
+            return
```

Comment:
I don’t think that is good solution. We should return number value in any case. If we can’t get value the exception should be raised